tests/pytest_plugins/db.py

```diff
@@ -0,0 +1,22 @@
+from os import environ
+
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from report.db import Base
+
+
+@pytest.fixture(scope="session")
+def db_engine():  # pragma: no cover
+    engine = create_engine(environ["DATABASE_URL"])
+
+    Base.metadata.drop_all(engine)
+    Base.metadata.create_all(engine)
+
+    return engine
+
+
+@pytest.fixture(scope="session")
+def db_sessionfactory():  # pragma: no cover
+    return sessionmaker()
```

Comment:
Report [overrides the `db_engine` fixture with its own](https://github.com/hypothesis/report/blob/d0366d5d9c36a556b87dc61994567de3c391edf0/tests/conftest.py#L29-L34) and doesn't use the `db_sessionfactory` fixture.